src/porepy/numerics/time_step_control.py

```diff
@@ -393,7 +397,7 @@ def __repr__(self) -> str:
         return s
 
     def next_time_step(
```

Comment:
I find the interpretation of "next_time_step" ambiguous at best. Wouldn't "compute_time_step" be more precise?

Reply:
Agree. I will rename the method.